bson-scala/src/test/scala/org/mongodb/scala/bson/codecs/MacrosSpec.scala

```diff
@@ -530,6 +530,7 @@ class MacrosSpec extends BaseSpec {
   }
 
   it should "support tagged types in case classes" in {
+    assume(!scala.util.Properties.versionNumberString.startsWith("2.11"))
```

Comment:
This is failing to compile with Scala 2.11 - not sure how it worked with the previous plugin as its a scala error. Its possible that we are using a newer scala 2.11 version patch which highlights this error.

As Scala 2.11 has been EOL for a long time so I'm not sure its worth investigating further.